examples/ecs/ambiguity_checker.rs

```diff
@@ -0,0 +1,63 @@
+use bevy::{ecs::schedule::ReportExecutionOrderAmbiguities, log::LogPlugin, prelude::*};
+
+fn main() {
+    App::new()
+        .add_plugin(LogPlugin)
+        .insert_resource(MyStartupResource(0))
+        // This resource allows to control how Ambiguity Checker will report unresolved ambiguities.
+        // By default only a warning with the amount of unresolved ambiguities is shown, but
```

Comment:
```suggestion
        // By default only a warning with the number of unresolved ambiguities is shown, but
```

English is dumb, sorry.